### PR TITLE
Update/0.14.1

### PR DESCRIPTION
**Pull request by Dominion5254**

### CODE REVIEW

File: Dockerfile
Comment (k0gen) [inline Dockerfile:1]: While it's technically feasible to mix Debian and Alpine build stages, it's important to do so with a clear understanding of the potential complexities and trade-offs. Try to maintain consistency in your choice of base images throughout your multi-stage build to simplify the process and minimize compatibility concerns.

File: Dockerfile [hunk 1]
```diff
@@ -32,13 +35,16 @@ RUN npm prune --production --legacy-peer-deps
...
-ARG PLATFORM
 
-RUN apt update
-RUN apt install -y bash curl iproute2 wget
+RUN apk update && \
+  apk add --no-cache \
+  bash \
+  curl \
+  iproute2 \
+  yq
```
Comment (k0gen) [inline Dockerfile]: add tini

File: Dockerfile [hunk 2]
```diff
@@ -1,13 +1,16 @@
 # ---------------
 # Install Dependencies
 # ---------------
-FROM node:16-stretch-slim as builder
+FROM node:16-buster-slim as builder
 
 ARG PLATFORM
 
 ADD https://github.com/krallin/tini/releases/download/v0.19.0/tini-static-${PLATFORM} /tini
 RUN chmod +x /tini
```
Comment (k0gen) [inline Dockerfile]: can be removed if you install it in runner stage

File: Dockerfile [hunk 3]
```diff
@@ -32,13 +35,16 @@ RUN npm prune --production --legacy-peer-deps
...
 # ---------------
-FROM node:16-stretch-slim as runner
+FROM node:16-alpine as runner
 
 ARG ARCH
-ARG PLATFORM
 
-RUN apt update
-RUN apt install -y bash curl iproute2 wget
+RUN apk update && \
```
Comment (k0gen) [inline Dockerfile]: you don't need `apk update`

**Review by k0gen — CHANGES_REQUESTED** [review]

File: Dockerfile
Comment (Dominion5254) [reply Dockerfile:1]: Updated builder to Alpine

File: Dockerfile (hunk 1 above)
Comment (Dominion5254) [reply Dockerfile]: Updated

File: Dockerfile (hunk 2 above)
Comment (Dominion5254) [reply Dockerfile]: Updated

File: Dockerfile (hunk 3 above)
Comment (Dominion5254) [reply Dockerfile]: Updated

**Review by k0gen — APPROVED** [review]
🛳️ it

**Pull request merged**